sdk/communication/Azure.Communication.PhoneNumbers/src/SipRouting/SipTrunkRoute.cs

```diff
@@ -0,0 +1,59 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text.Json;
+using Azure.Core;
+
+namespace Azure.Communication.PhoneNumbers.SipRouting
+{
+    [CodeGenClient("TrunkRoute")]
+    [CodeGenSuppress("SipTrunkRoute", typeof(string), typeof(string))]
+    public partial class SipTrunkRoute
+    {
+        /// <summary> Initializes a new instance of <see cref="SipTrunkRoute"/>. </summary>
+        /// <param name="name"> Gets or sets name of the route. </param>
```

Comment:
Constructor parameters does not get the value. This param description does not seem accurate.

Reply:
Fixed comments.